apis/projectcontour/v1alpha1/httpproxy.go

```diff
@@ -36,18 +36,30 @@ type Include struct {
 	Name string `json:"name"`
 	// Namespace of the HTTPProxy
 	Namespace string `json:"namespace,omitempty"`
-	// Condition is a set of routing properies that is applied to an HTTPProxy in a namespace.
-	Condition `json:"conditions"`
+	// Conditions are a set of routing properties that is applied to an HTTPProxy in a namespace.
+	Conditions []Condition `json:"conditions"`
 }
 
 // Condition are policies that are applied on top of HTTPProxies.
 type Condition struct {
 	// Prefix defines a prefix match for a request.
 	Prefix string `json:"prefix,omitempty"`
-	// HeadersMatch represent a set of HTTP headers that match the key/value exactly as specified.
-	HeadersMatch map[string][]string `json:"headersMatch,omitempty"`
-	// HeadersContain represent a set of HTTP headers that match the key exactly and the value as a contains.
-	HeadersContain map[string][]string `json:"headersContain,omitempty"`
+	// Header matches only the key of the header ignoring any value supplied in the request.
+	HeaderOnly string `json:"headerOnly,omitempty"`
```

Comment:
Maybe HeaderPresent?

Reply:
I like that better and it's what Envoy calls it. 👍